source/assets/stylesheets/styles/_jobs_page_tile.scss

```diff
@@ -110,3 +110,10 @@
     min-height: 106px;
   }
 }
+
+.jobs-page-tile-no-roles__paragraph {
```

Comment:
We should stick to the block name in the file for this rule (or make a new file for the new block). How do you feel about `.jobs-page-tile__no-roles-paragraph`?